platform/posix/transport/src/openssl_posix.c

```diff
@@ -746,13 +746,17 @@ int32_t Openssl_Recv( NetworkContext_t * pNetworkContext,
         /* Set the file descriptor for poll. */
         pollFds.fd = pOpensslParams->socketDescriptor;
 
+        /* #SSL_pending returns a value > 0 if application data
+         * from the last processed TLS record remains to be read. */
         if( ( bytesToRecv > 1 ) || ( SSL_pending( pOpensslParams->pSsl ) > 0 ) )
         {
             shouldRead = 1U;
         }
         else
         {
-            /* Non-speculative read so check if there is data to read from the socket. */
+            /* Speculative read for the start of a payload.
+             * Note: This is done to avoid blocking when no
+             * data is available to be read from the socket. */
```

Comment:
Formatting of comment is not aligned to the code
```suggestion
           /* Speculative read for the start of a payload.
            * Note: This is done to avoid blocking when no
            * data is available to be read from the socket. */
```

Reply:
That is already aligned. What you are suggesting won't pass formatting checks..